storybook/src/components/button/button.css-stories.ts

```diff
@@ -22,4 +22,4 @@ const { Primary, PrimaryCompact, Secondary, Tertiary } = buttonStories({
   },
 });
 
-export { Primary, PrimaryCompact, Secondary, Tertiary };
+export { Map, Primary, PrimaryCompact, Secondary, Tertiary };
```

Comment:
Ik had 'm eerst aan het einde gezet, maar toen ging ESLint klagen 😐 Exports moeten ook gesorteerd zijn.

Wat denken jullie: Voor nu zo laten en later structurele oplossing bedenken of voor deze regel uitzetten?